cmd/node/config/enableEpochs.toml

```diff
@@ -108,6 +108,9 @@
     # BuiltInFunctionOnMetaEnableEpoch represents the epoch when built in function processing on metachain is enabled
     BuiltInFunctionOnMetaEnableEpoch = 5
 
+    # ESDTNFTCreateOnMultiShard represents the epoch when esdt nft creation is enabled on multiple shards
+    ESDTNFTCreateOnMultiShard = 6
```

Comment:
please add `EnableEpoch` suffix in order to match the rest of the values from the configuration

Reply:
done